src/pkg/build/assembler.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package build
+
+import (
+	"fmt"
+)
+
+// validAssemblers contains of list of know Assemblers
+var validAssemblers = map[string]bool{
+	"SIF":     true,
+	"sandbox": true,
+}
+
+// Assembler is responsible for assembling an image from a bundle
```

Comment:
Can you add some comments here giving an example of what an assembler might need to do? Without some context it's not clear for people coming into the code what the difference is between bundle and image.